RELEASES.md

```diff
@@ -85,12 +85,24 @@ soci-snapshotter-$VERSION-linux-amd64.tar.gz
     THIRD_PARTY_LICENSES
     NOTICE.md
 soci-snapshotter-$VERSION-linux-amd64.tar.gz.sha256sum
+soci-snapshotter-$VERSION-linux-amd64-static.tar.gz
+    soci-snapshotter-grpc
+    soci
+    THIRD_PARTY_LICENSES
+    NOTICE.md
```

Comment:
Two repetitions of this was ok, but I think four is pushing it, and if we add more it will be too much. Can we instead describe the consistent contents of the tarballs separately in a single place?